analyses/chromosomal-instability/01b-visualization-cnv-sv.Rmd

```diff
@@ -307,7 +304,7 @@ circos_map_plot(
 ```{r}
 circos_map_transloc(transloc_df,
   add_track = FALSE, # We change this to true to add on to our already existing plot
-  sample_names = samples_for_examples[1],
+  sample_names = sample(transloc_df$biospecimen_id1, 1),
```

Comment:
I had to do this because not all samples have BNDs so this will kick back an error.